schema/operation.json

```diff
@@ -3,26 +3,32 @@
     "$id": "http://standards.ncats.io/operation.json",
     "anyOf": [
         {
-            "$ref": "#/definitions/OperationFilterResults"
+            "$ref": "#/definitions/OperationFilterKgraphOrphans"
```

Comment:
These appear to have been reordered. We should be able to make sure that they are sorted so that our PRs don't always look like this. I'll add an issue to fix the schema generation code.